osbs/api.py

```diff
@@ -610,9 +581,8 @@ def create_worker_build(self, **kwargs):
         if kwargs.get('platforms'):
             raise ValueError("Worker build called with unwanted platforms param")
 
-        arrangement_version = kwargs['arrangement_version']
         kwargs.setdefault('inner_template', WORKER_INNER_TEMPLATE.format(
-            arrangement_version=arrangement_version))
+            arrangement_version=DEFAULT_ARRANGEMENT_VERSION))
```

Comment:
Should we maybe rename the templates from `*_inner:6.json` to `*_inner.json`? Or straight up remove them since they will not be used anymore?

Reply:
removal will be done in your PR, I don't want to rename it just for sake of removing that 1 var

Reply:
OK, I guess I can remove a few leftovers afterwards